torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19515,6 +19515,29 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            sample_inputs_func=sample_inputs_squeeze_multiple),
+    OpInfo('squeeze_copy',
+           ref=_squeeze_ref,
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16, torch.chalf),
+           supports_out=True,
+           assert_autodiffed=True,
+           autodiff_fusible_nodes=[],  # aliases inputs, shouldn't be fused
+           autodiff_nonfusible_nodes=[],  # aliases inputs, shouldn't be fused
```

Comment:
There's no alias here so this is likely wrong.

Reply:
Deleted. I put these in several other `_copy` `OpInfo`s, should I remove them?